Net07.HW/Net07.DynamicProgrammingAndClasses/Net07.DynamicProgrammingAndClasses/Circle.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Net07.DynamicProgrammingAndClasses
+{
+    class Circle : Shape
+    {
+        public double Radius { get; }
+        public Point Center { get; }
+        public Circle(Point point)
```

Comment:
Not sure if we can create Circle without a radius.
We can create a few constructors when there some not mandatory parameters. But in the case of `Circle`, all parameters are mandatory and should always be initialized